### DISCUSSION TITLE
MAIA novelty detection only detects image labeling

### DISCUSSION THREAD

**Discussion by Hermann3010** [biigle/core#874] [Q&A] [answered]
Hi,
results of novelty detection only give me back the image labeling (what is somehow clear ;)
When cutting away the labeling away I have got the following error: 

"New MAIA jobs cannot be created for volumes that contain images smaller than 512 pixels on one edge."

...is it now impossible to run MAIA with that images? Or can I solve the problem with the novelty detection settings (I have tried to change some settings, but without success).

Happy about any help!!!  

![Bild1](https://github.com/biigle/core/assets/82365322/ecca32a1-0a1e-4690-9e72-31706fa5a0ef)

**Comment by mzur** [comment, marked as answer]
Hi, supporting smaller images is an issue [we want to address](https://github.com/biigle/maia/issues/28) at some point but currently these are not supported. So novelty detection will not work with these images, one way or the other, sorry.

But you can still use MAIA if you skip novelty detection and provide manual annotations as training samples. For this you can start annotating your volume until you are quite certain that you have some training samples for each of the object classes you are interested in. Then you can start a new MAIA job with the "existing annotations" option. MAIA will still run on the annotated images so you might end up with duplicate annotations there. But these could also give you an idea how well the MAIA detection worked.

Another way could be to use training annotations from a different volume. This is possible with the "knowledge transfer" option. However, this option currently requires existing area metadata for each image in both volumes (a requirement that we want to [remove at some point](https://github.com/biigle/maia/issues/126), too). The metadata may be there if your images provide it or you could also upload "fake" metadata with an area of 1 for each image. This effectively disables the "scaling" aspect of knowledge transfer but the method might still work.

**Reply by Hermann3010** [reply]
Ah, ok!!!
Thanks al lot for the quick reply!!!!!
